src/AppInstallerCLICore/Argument.cpp

```diff
@@ -97,6 +97,8 @@ namespace AppInstaller::CLI
             return Argument{ "wait", NoAlias, Args::Type::Wait, Resource::String::WaitArgumentDescription, ArgumentType::Flag, false };
         case Args::Type::ProductCode:
             return Argument{ "product-code", NoAlias, Args::Type::ProductCode, Resource::String::ProductCodeArgumentDescription, ArgumentType::Standard, false };
+        case Args::Type::IgnoreWarnings:
+            return Argument{ "ignore-warnings", NoAlias, Args::Type::IgnoreWarnings, Resource::String::IgnoreWarningsArgumentDescription, ArgumentType::Flag, false };

```

Comment:
nit: I think "ignore-install-warnings" would be more accurate, "ignore-warnings" seems too generic. Also please update the variable name to leave room for future warnings

Reply:
> nit: I think "ignore-install-warnings" would be more accurate, "ignore-warnings" seems too generic. Also please update the variable name to leave room for future warnings

Also you'll need to expose them in each applicable command.

Reply:
I've seen a couple of features reported with similar concerns like:
* #715

I just added it to:
* #2128

Reply:
Breaking or not, we should change `--force` per this discussion: #2112